packages/affiliates/libs/start.js

```diff
@@ -0,0 +1,19 @@
+// This file is a common bootstrapping library to initialize configs and web3 apps in the /apps dir.
+// It expects a function to be passed in. This file is repsonsible for constructing the configuration
+// file and other dependencies for that function and injecting it. The idea is that any app would share
+// the same pattern for getting configuration and dependencies.
+// example: see how its used in apps/DeployerRewards
+require("dotenv").config();
+const assert = require("assert");
+const Web3 = require("web3");
+const Path = require("path");
+const [, , configPath] = process.argv;
+
+assert(configPath, "Requires relative path to config params");
+const config = require(Path.join(process.cwd(), configPath));
+assert(process.env.CUSTOM_NODE_URL, "requires CUSTOM_NODE_URL");
+
+module.exports = async App => {
+  const web3 = new Web3(new Web3.providers.HttpProvider(process.env.CUSTOM_NODE_URL));
```

Comment:
nit: can we use `getWeb3` from https://github.com/UMAprotocol/protocol/blob/master/packages/common/src/ProviderUtils.js?

Reply:
I think i was using getWeb3 but found it confusing to reason about when doing testing without truffle and/or switching networks. I don't actually remember the exact reason. @chrismaree suggested I copy what he does in another script, though i am open to whatever you think is the best here.

Reply:
What issues did you have with it? Could we improve that function rather than using a custom solution? IIRC, the reason @chrismaree used this method was that we hadn't written `getWeb3()` yet.

The point of `getWeb3()` is to:
1. Mirror the way truffle resolved networks when not using truffle (i.e. still using our truffle config).
2. Detect when running within truffle/truffle test and just fall back to the truffle web3 in those cases.

Reply:
I agree with @mrice32 that we should ideally use `getWeb3` everywhere. It's already configured in the `affiliates` script [here](https://github.com/UMAprotocol/protocol/blob/4bf7ae5006390c8030d62b887e6a71311d8871c4/packages/affiliates/libs/affiliates.js#L9). I think ideally the main entry point (be it this `start.js` or a set of unit tests) should inject their respective web3 object configured for the appropriate network.